src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -104,6 +108,10 @@ public Drivetrain() {
     SmartDashboard.putNumber("driveVelocity", 0);
   }
 
+  public ChassisSpeeds getSpeeds() {
+    return m_speeds;
```

Comment:
where is m_speeds being updated? in drivetrain it updates in periodic, but I don't see that here, maybe just return the robot relative speeds straight from the odometry